cpp/include/cudf/dictionary/dictionary_factories.hpp

```diff
@@ -43,6 +43,8 @@ namespace cudf
  * d is now {["a","c","d"],[1,0,0,2,2]}
  * ```
  *
+ * The null_mask and null count for the output column are copied from the indices column.
```

Comment:
Maybe add a null element to the example above.

Reply:
What would that look like? Something like this?
```
 k = ["a","c","d"]
 i = [1,0,null,2,2]
 d = make_dictionary_column(k,i)
 d is now {["a","c","d"],[1,0,unk,2,2]} bitmask={1,1,0,1,1}
 ```
I don't want to imply the indices have a null bitmask.

Reply:
sure, that looks good.